src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { Provider } from 'react-redux';
+
+import MainPage from './components/MainPage/MainPage.jsx';
+import ErrorPage from './components/ErrorPage/ErrorPage.jsx';
+import NotFoundPage from './components/NotFoundPage/NotFoundPage.jsx';
+import BalancePage from './components/BalancePage/BalancePage.jsx';
+
+import store from './redux/store.js';
+
+const App = () => (
+  <Provider store={store}>
+    <Router>
+      <Switch>
+        <Route exact path="/">
```

Comment:
could you create a separate file like `routes.js` where save all of the existing routes
```
export const HOME = '/';
export const BALANCE = '/balance';
...
```
then import it to this file
```
import * as routes from 'src/navigation/routes';
```
and use values here like for example, `routes.HOME`, please?

Reply:
Fixed. Routes are now variables